docs/source/semi_ring.py

```diff
@@ -50,19 +52,19 @@ def __add__(self, other: "SemiRingDict[K, V]") -> "SemiRingDict[K, V]":
 
 @defop
 def Sum(
-    e1: Annotated[SemiRingDict[K, V], Scoped(1)],
-    k: Annotated[Operation[[], K], Bound(0)],
-    v: Annotated[Operation[[], V], Bound(0)],
-    e2: Annotated[SemiRingDict[S, T], Scoped(0)],
+    e1: SemiRingDict[K, V],
+    k: Annotated[Operation[[], K], Scoped[A]],
+    v: Annotated[Operation[[], V], Scoped[A]],
+    e2: Annotated[SemiRingDict[S, T], Scoped[A]],
 ) -> SemiRingDict[S, T]:
     raise NotImplementedError
 
 
 @defop
 def Let(
-    e1: Annotated[T, Scoped(1)],
-    x: Annotated[Operation[[], T], Bound(0)],
-    e2: Annotated[S, Scoped(0)],
+    e1: Annotated[T, Scoped[A]],
+    x: Annotated[Operation[[], T], Scoped[B]],
+    e2: Annotated[S, Scoped[B]],
 ) -> S:
```

Comment:
Should this be `Annotated[S, Scoped[A]]`?

Reply:
Or maybe `e1` should not be annotated at all.

Reply:
Changed to `Annotated[S, Scoped[A]]`.